software/validation/stellar_density_control_arrays.py

```diff
@@ -0,0 +1,115 @@
+"""
+This file reads in a list of names of the raw csv catalogs provided
+by Dave Monet, reads in those files, assembles them into HEALPIX maps
+segregated by magnitude, and outputs those maps as text files to be
+read in by another script and plotted.
+"""
+
+from __future__ import with_statement
+
+import numpy as np
+import healpy as hp
+import os
+
+import time
+import sys
+
+if __name__ == "__main__":
+
+    if len(sys.argv) != 3:
+        print "Need to pass in a to do list and a job dex"
+        # job dex is an integer appended to the end of the output
+        # files produced by this script so they don't collide with
+        # the output files produced by another script.  This is done
+        # so that you can run many jobs in parallel and get through
+        # the whole catalog faster.
+
+
+    to_do_list_file = sys.argv[1]
+    job_dex = int(sys.argv[2])
+    print to_do_list_file, job_dex
+
+    t_start = time.time()
+
+    NSIDE = 64
```

Comment:
As in the last file, why is this hard-coded to 64? I understand wanting reproducible results for validation scripts, but a comment would help someone who copies this script to use as a starting point for writing their own.